### PR TITLE
Session 1 Assignment

### PR DESCRIPTION
**Pull request by angielynr**
- Created local db -> connected local db to application
- Creation of migration scripts
- Filled in to do's for filtering, SingleOrDefault, Loading related data manually, explicitly & eagerly
session1

### CODE REVIEW

**Review by JJRimorin — APPROVED** [review]
LGTM! Please wait for Lai's review before merging. Thank you!

**Review by laira-salvador — APPROVED** [review]
Nicely done!

**Pull request merged**
